RELEASE.md

```diff
@@ -1,28 +1,40 @@
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
-mlkem-native alpha
+mlkem-native v1.0.0-beta
 ==================
 
 About
 -----
 
-mlkem-native is a C90 implementation of [ML-KEM](https://doi.org/10.6028/NIST.FIPS.203) targeting
-PC, mobile and server platforms. It is a fork of the ML-KEM [reference
-implementation](https://github.com/pq-crystals/kyber/tree/main/ref).
+mlkem-native is a secure, fast and portable C90 implementation of [ML-KEM](https://doi.org/10.6028/NIST.FIPS.203).
+It is a fork of the ML-KEM [reference implementation](https://github.com/pq-crystals/kyber/tree/main/ref).
 
-mlkem-native aims to be fast, secure, and easy to use: It provides native code backends in C, AArch64 and
-x86_64, offering state-of-the-art performance on most Arm, Intel and AMD platforms. The C code in [mlkem/*](mlkem) is
-verified using [CBMC](https://github.com/diffblue/cbmc) to be free of undefined behavior. In particular, there are no
-out of bounds accesses, nor integer overflows during optimized modular arithmetic.
+mlkem-native includes native backends for AArch64 and AVX2, offering competitive performance on most Arm, Intel and AMD platforms
+(see [benchmarks](https://pq-code-package.github.io/mlkem-native/dev/bench/)). The frontend and the C backend (i.e., all C code in [mlkem/*](mlkem) and [mlkem/fips202/*](mlkem/fips202)) are verified
+using [CBMC](https://github.com/diffblue/cbmc) to be free of undefined behaviour. In particular, there are no out of
+bounds accesses, nor integer overflows during optimized modular arithmetic.
+
+mlkem-native is supported by the [Post-Quantum Cryptography Alliance](https://pqca.org/) as part of the [Linux Foundation](https://linuxfoundation.org/).
 
 Release notes
 =============
 
-This is first official release of mlkem-native, a C90 implementation of [ML-KEM](https://doi.org/10.6028/NIST.FIPS.203) targeting
-PC, mobile and server platforms.
-This alpha release of mlkem-native features complete backends in C, AArch64 and x86_64, offering state-of-the-art performance on most Arm, Intel and AMD platforms.
+This is second official release of mlkem-native, a C90 implementation of [ML-KEM](https://doi.org/10.6028/NIST.FIPS.203) targeting PC, mobile and server platforms.
+This beta release of mlkem-native features complete backends in C, AArch64 and x86_64, offering state-of-the-art performance on most Arm, Intel and AMD platforms.
+
+What's New
+----------
+
+Compared to [v1.0.0-alpha](https://github.com/pq-code-package/mlkem-native/releases/tag/v1.0.0-alpha) the following
+major improvements have been integrated into mlkem-native:
```

Comment:
Did we have complete CBMC proofs in v1.0.0-alpha?

Reply:
Good point - only had coverage of `mlkem/*` in the first release. I'll add that.